apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -586,14 +586,13 @@ struct ChatView: View {
 
         private func cancelFileUIAction(_ fileId: Int64) -> UIAction {
             UIAction(
-                title: NSLocalizedString("Cancel", comment: "chat item action"),
-                image: UIImage(systemName: "xmark"),
-                attributes: [.destructive]
+                title: NSLocalizedString("Stop file", comment: "chat item action"),
```

Comment:
I haven't called it Stop because you can cancel before transfer is started, there's nothing to stop yet.

Reply:
At least receive is not showing this button in RcvInvitation status, only Delete. Maybe exclude SndStored as well? Then at least it makes more sense as it's only for "in progress" statuses.

Reply:
> I haven't called it Stop because you can cancel before transfer is started, there's nothing to stop yet.

"Stop" also means "prevent something from happening", not necessarily "terminate" something that is already happening": https://dictionary.cambridge.org/dictionary/english/stop. So stop is clear enough in this context, and also shorter.

> Then at least it makes more sense as it's only for "in progress" statuses.

It's ok to use "stop" for something due to start.

Reply:
"Cancel" was very unclear. "Cancel file" is an option, but it is less clear what it means, I think